compiler/dyno/util/lint

```diff
@@ -17,15 +17,17 @@ from typing import Set, Dict, NamedTuple, Tuple, List
 #   * currently does not handle inherited members (it doesn't see them at all)
 #   * currently does not handle methods not declared in the class definition (though maybe it does sometimes? Need more investigation)
 
-class LintError(NamedTuple):
-    klass: str
-    method: str
-    not_mentioned: Tuple[str]
+# class LintError(NamedTuple):
```

Comment:
I'm seeing a lot of commented-out stuff in this file. Probably better to remove it and leave it preserved in commit history.